client/src/App.vue

```diff
@@ -230,8 +230,6 @@ export default class App extends Mixins(SessionTimerMixin) {
 #main-column
```

Comment:
I can't add a comment on the favicon.ico file so I'll do it here.

Please use a favicon PNG file from our UI projects as it has a transparent background.

Reply:
Good call will do, i thought it looked off, thanks!